elasticpress-stream.php

```diff
@@ -4,16 +4,16 @@
  * Plugin URI:        https://github.com/10up/elasticpress-stream
  * Description:       Use ElasticPress to power Stream with Elasticsearch.
  * Version:           1.0.0
- * Requires at least: 
- * Requires PHP:      
+ * Requires at least:
+ * Requires PHP:
```

Comment:
@jeffpaul probably better to make these reflect the minimum requirements of ElasticPress, right? If so, @g-kanoufi, these can be set as
```
 * Requires at least: 5.6
 * Requires PHP:      7.0
```

Reply:
You can now have that be `Requires Plugins: elasticpress` which should work for WP 6.5+